index.js

```diff
@@ -106,18 +106,24 @@ function math_block(state, start, end, silent){
         max = state.eMarks[start]
 
     if(pos + 2 > max){ return false; }
-    if(state.src.slice(pos,pos+2)!=='$$'){ return false; }
+
+    /** @type {readonly ['$$', '\\['} */
```

Comment:
are type annotations accepted? Should I remove them?

Reply:
T

Reply:
T